### DISCUSSION TITLE
drag and drop with pagination

### DISCUSSION THREAD

**Discussion by Todomed** [rappasoft/laravel-livewire-tables#574] [Feature Requests]
It is a good option to be able to change the position of the rows with pagination. Currently this function freezes when there are many records and these are not paged.

**Comment by rappasoft** [comment]
[It's literally in the docs as to why](https://rappasoft.com/docs/laravel-livewire-tables/v1/rows/drag-and-drop#a-warning-about-reordering)

**Reply by Todomed** [reply]
thanks for your prompt response
